### ISSUE TITLE
gnn baseline seems not working 

### ISSUE THREAD

**Issue by smallbox120** [hytseng0509/CrossDomainFewShot#1]
the loss seem to converge on bad state, even after the first few epochs and the validation result is equivalent to random guessing. And I waited about 40 epochs, seems to me that it won't work even if I train a few hundred more epochs. I train the model from scratch instead of pretrained parameters.

**Comment by hytseng0509** [comment]
The GNN model we use cannot be trained without the pre-trained feature encoder.
You can refer to the original implementation [here](https://github.com/vgsatorras/few-shot-gnn) for training the model from scratch.

**Comment by hytseng0509** [comment]
It is not tricky to train the GNN method from scratch. The code we provide here can only train the model with the pre-trained feature encoder. You can refer to the original implementation [here](https://github.com/vgsatorras/few-shot-gnn) for training the model from scratch.
